packages/scoutgame-ui/src/components/scout/ScoutPage.tsx

```diff
@@ -71,8 +73,8 @@ export async function ScoutPage({
             gap: 2
           }}
         >
-          <Suspense fallback={<LoadingCards count={3} />}>
-            <ScoutPageCarousel />
+          <Suspense key={carousel} fallback={<LoadingCards count={3} />}>
```

Comment:
we shouldn't need to re-query the data every time you change tabs. ideally it just loads once on page load.

Reply:
Yep. I updated this and the async function removal so it doesn't load again